### PR TITLE
Migrate layouts styles to emotion and remove tailwind

### PR DESCRIPTION
**Pull request by mwolman**
#### :page_facing_up: Description:

* Migrate AuthLayout styles to emotion
* Migrate MainLayout styles to emotion
* Remove Tailwind and update all the files that had references to it

### CODE REVIEW

**Comment by mconiglio** [comment]
:popa:

File: postcss.config.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 /* eslint-disable global-require */
```
Comment (mconiglio) [inline postcss.config.js:1]: do we need postcss at all? 🤔

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,351 @@
+html {
```
Comment (mconiglio) [inline src/styles.css:1]: why did you add this file? and all these styles also?

File: src/layouts/MainLayout/MainLayout.js [hunk 3]
```diff
@@ -3,16 +3,18 @@ import PropTypes from 'prop-types';
...
-    </main>
-  </>
-);
+const MainLayout = ({ children, noHeader }) => {
+  const Component = noHeader ? MainContainerWithoutHeader : MainContainer;
+
+  return (
+    <>
+      {!noHeader && <Navbar />}
+      <Component>{children}</Component>
```
Comment (mconiglio) [inline src/layouts/MainLayout/MainLayout.js:14]: why don't you just define a `MainContainer` styled component that receives a prop called "withHeader" and set the styles conditionally in the styles file? that's one of the biggest advantages of css-in-js, to extract all the styling logic to it

File: src/components/Navbar/NavBar.styles.js [hunk 4]
```diff
@@ -1,6 +1,13 @@
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 
+// If you change the height of this element, go to:
```
Comment (mconiglio) [inline src/components/Navbar/NavBar.styles.js:4]: wouldn't it be better to just put the header height in the theme? this kind of comments shouldn't be necessary

File: postcss.config.js (hunk 1 above)
Comment (lucaleivaloop) [reply postcss.config.js:1]: I don't think so.

Comment (mwolman) [reply postcss.config.js:1]: Good, I'm going to delete it, thanks 🤗

File: src/components/Navbar/NavBar.styles.js [hunk 5]
```diff
@@ -1,13 +1,20 @@
...
```
Comment (Rminetti) [inline src/components/Navbar/NavBar.styles.js:4]: I see that you marked the conversation about this comment being removed as resolved, but I don't understand how it is resolved? Did you agree to leave the comment?

**Comment by mconiglio** [comment]
Could you please remove `postcss-cli` from the dependencies?

**Comment by mwolman** [comment]
> Could you please remove `postcss-cli` from the dependencies?

Sure 😎

File: src/components/App/App.styles.js [hunk 6]
```diff
@@ -5,5 +5,358 @@ export const loadingStyles = css`
...
+    vertical-align: middle;
+  }
+
+  img,
+  video {
+    max-width: 100%;
+    height: auto;
+  }
+
+  @keyframes spin {
```
Comment (mconiglio) [inline src/components/App/App.styles.js:316]: could you remove these @keyframes that we are for sure not using them?

File: src/layouts/AuthLayout/AuthLayout.styles.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const AuthContainer = styled.main`
+  padding-top: ${({ noHeader, theme }) =>
+    noHeader ? '' : `${theme.height.header}`};
```
Comment (seedwalk) [inline src/layouts/AuthLayout/AuthLayout.styles.js:5]: isn't this declaring `padding-top` but leaving it without any value ?

 si `noHeader === true`
```padding-top: ;```

File: src/layouts/MainLayout/MainLayout.styles.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const MainContainer = styled.main`
+  padding-top: ${({ noHeader, theme }) =>
+    noHeader ? '' : `${theme.height.header}`};
```
Comment (seedwalk) [inline src/layouts/MainLayout/MainLayout.styles.js:5]: same here

File: src/layouts/AuthLayout/AuthLayout.styles.js (hunk 7 above)
Comment (mwolman) [reply src/layouts/AuthLayout/AuthLayout.styles.js:5]: Yes 🤦🏻‍♀️. I'm gonna change it to `null` and emotion will remove it 💪🏻  thanks!
